generators/client/templates/vue/src/main/webapp/app/shared/config/formatter.ts.ejs

```diff
@@ -0,0 +1,122 @@
+function warn(msg: string, err?: Error) {
```

Comment:
indent with 2 spaces instead of 4, for all this file

Reply:
as this file comes from somewhere, is there a licence issue ?

Reply:
It is MIT license: https://github.com/kazupon/vue-i18n/blob/dev/LICENSE